packages/plugin-graphql-client/loader.js

```diff
@@ -0,0 +1,13 @@
+function applyGraphqlLoader(config) {
+  config.module.rules.push({
+    test: /\.(graphql|gql)$/,
+    exclude: /node_modules/,
+    loader: 'graphql-tag/loader',
```

Comment:
missing dependencies in package.json

Reply:
`graphql-tag` is already in `package.json`